openmetadata-ui/src/main/resources/ui/src/utils/SSOUtils.ts

```diff
@@ -0,0 +1,129 @@
+/*
+ *  Copyright 2025 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import LdapIcon from '../assets/img/ic-ldap.svg';
+import SamlIcon from '../assets/img/ic-saml.svg';
+import Auth0Icon from '../assets/img/icon-auth0.png';
+import CognitoIcon from '../assets/img/icon-aws-cognito.png';
+import AzureIcon from '../assets/img/icon-azure.png';
+import GoogleIcon from '../assets/img/icon-google.png';
+import OktaIcon from '../assets/img/icon-okta.png';
+import SSOIcon from '../assets/svg/sso-settings.svg';
+
+import { AuthProvider } from '../generated/settings/settings';
+
+export interface ProviderOption {
+  key: AuthProvider;
+  label: string;
+  icon: string;
+}
+
+/**
+ * Get the display name for an authentication provider
+ * @param provider - The authentication provider
+ * @returns The human-readable display name
+ */
+export const getProviderDisplayName = (provider: string): string => {
+  switch (provider) {
+    case AuthProvider.Azure:
+      return 'Azure AD';
+    case AuthProvider.Google:
+      return 'Google';
+    case AuthProvider.Okta:
+      return 'Okta';
+    case AuthProvider.Auth0:
+      return 'Auth0';
+    case AuthProvider.AwsCognito:
+      return 'AWS Cognito';
+    case AuthProvider.Saml:
+      return 'SAML';
+    case AuthProvider.CustomOidc:
+      return 'Custom OIDC';
+    case AuthProvider.Basic:
+      return 'Basic Authentication';
+    default:
+      return provider?.charAt(0).toUpperCase() + provider?.slice(1);
+  }
+};
+
+/**
+ * Get the icon for an authentication provider
+ * @param provider - The authentication provider
+ * @returns The icon component or null
+ */
+export const getProviderIcon = (provider: string): string | null => {
+  switch (provider) {
+    case AuthProvider.Azure:
+      return AzureIcon;
+    case AuthProvider.Google:
+      return GoogleIcon;
+    case AuthProvider.Okta:
+      return OktaIcon;
+    case AuthProvider.Auth0:
+      return Auth0Icon;
+    case AuthProvider.AwsCognito:
+      return CognitoIcon;
+    case AuthProvider.LDAP:
```

Comment:
I think we are not using it at all the place

Reply:
Used in: SettingsSso.tsx, SSOConfigurationForm.tsx, SSODocPanel.tsx